### DISCUSSION TITLE
Qodana Working/Project Directory Property Issue

### DISCUSSION THREAD

**Discussion by vizniuk** [JetBrains/Qodana#87] [Q&A] [answered]
There is currently an issue with latest build of Qodana (including plugin to TeamCity or standalone Docker image).
When using **Working/Project Directory** attribute in CLI or Docker generated UI no longer picks up Origin on Project attribute, so feature to **"Open in IDE" no longer works**.
Is this a known issue or there are some workaround ?
Thanks in advance.

**Comment by brichbash** [comment, marked as answer]
Hello,

Sorry for the delay.

The `Open in IDE` feature needs VCS data, and if you don't have it in a specified `working directory`, the feature won't work. You can replace `--project-dir` with `--source-directory` option, and in this case Qodana will inspect only the specified directory but still scan the project root.
